airbyte-webapp/craco.config.js

```diff
@@ -2,6 +2,14 @@
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 
 module.exports = {
+  jest: {
+    configure: (jestConfig) => {
+      return {
+        ...jestConfig,
+        setupFiles: [...jestConfig.setupFiles, "./__test__/globalSetup.ts"],
```

Comment:
Is `__test__` a good place for this to live? I could not find an existing alternative.

Reply:
Jest (as configured in CRA) already reads by default the `strc/setupTests.ts` file. I think we should move the setup code just into this file instead?